WeatherForecast/WeatherForecast/WeatherDetailView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  WeatherDetailView.swift
+//  WeatherForecast
+//
+//  Created by Tony Lim on 2/4/24.
+//
+
+import UIKit
+
+final class WeatherDetailView: UIView {
+    
+    private let iconImageView: UIImageView = UIImageView()
```

Comment:
```suggestion
    private let iconImageView: UIImageView = .init()
```
요건 취향인데 요렇게 쓰면 코드량이 조금 줄어여 ㅎㅎ

Reply:
안그래도 회사에서 리팩토링하면서 느꼈는데, 코드량이 많아져서 나중엔 .init으로 만들고 있더라구요. 감사합니다 ㅎㅎ